blockstack_client/constants.py

```diff
@@ -294,10 +295,10 @@
 # never changes, so safe to duplicate to avoid gratuitous imports
 MAXIMUM_NAMES_PER_ADDRESS = 25
 
-RPC_MAX_ZONEFILE_LEN = 4096     # 4KB
```

Comment:
Duplicate definitions will always eventually bite us. We need to set aside time to clean these kinds of things up (and probably come up with a strategy for tracking them), I think our exposure to problems like this are only likely to grow as constants expand outside this repo (into, say, blockstack.js, cli-blockstack, subdomain-registrar, etc.)

Reply:
Definitely.  Fortunately for us in this case, `blockstack_client` is completely gone in `develop`, and we're about to merge that to `master`.